src/towncrier/check.py

```diff
@@ -8,15 +8,11 @@
 
 import click
 
-from subprocess import CalledProcessError, check_output, STDOUT
+from subprocess import CalledProcessError, STDOUT
 
 from ._settings import load_config_from_options
 from ._builder import find_fragments
-
-
-def _run(args, **kwargs):
```

Comment:
I would prefer to keep the private `_run` and refactor it to always return decoded output and add a docstring that explains what it does.